samples/bluetooth/peripheral_hids_keyboard/src/main.c

```diff
@@ -851,10 +851,13 @@ static void num_comp_reply(bool accept)
 
 static void button_changed(uint32_t button_state, uint32_t has_changed)
```

Comment:
spelling mistake in commit title (buttom).

Reply:
Thanks. Fixed